recipes/wfdb/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "wfdb" %}
+{% set version = "1.3.5" %}
+{% set sha256 = "a8964ac7c4f0bb70a571bd4236e5324a6297a2a6b6cfbf62f057811295668cfb" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
```

Comment:
Consider adding `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Indeed that seems the toolbox is writtent in pure python.